crates/sui-axelar-cgp/move/sources/utils.move

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module axelar::utils {
+    use std::vector;
+
+    use sui::bcs;
+    use sui::hash;
+
+    /// Normalize last byte of the signature. Have it 1 or 0.
+    /// See https://tech.mystenlabs.com/cryptography-in-sui-cross-chain-signature-verification/
+    public fun normalize_signature(signature: &mut vector<u8>) {
+        // Compute v = 0 or 1.
+        let v = vector::borrow_mut(signature, 64);
```

Comment:
An assertion for the signature size could help a bit (make the error more explicit)